test/support/utils_sycl_defs.h

```diff
@@ -0,0 +1,66 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+// This file contains SYCL specific macros and abstractions
+// to support different versions of SYCL and to simplify its interfaces
+//
+// Functionality, required for the tests, is copied from sycl_defs.h and thus is not directly used
+// for compatibility with different versions of the library
+
+#ifndef _UTILS_SYCL_DEFS_H
+#define _UTILS_SYCL_DEFS_H
+
+#include <CL/sycl.hpp>
+
+// Combine SYCL runtime library version
+#if defined(__LIBSYCL_MAJOR_VERSION) && defined(__LIBSYCL_MINOR_VERSION) && defined(__LIBSYCL_PATCH_VERSION)
+#    define __LIBSYCL_VERSION                                                                                          \
```

Comment:
With such a name there is a risk of macro redefinition. What prefix do we use for other test-specific macros?

Reply:
We use `TEST_` prefix. I changed the name of this macro accordingly.